openmetadata-ui/src/main/resources/ui/src/components/Dashboard/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -622,172 +227,58 @@ const DashboardDetails = ({
-            onFeedUpdate={getEntityFeedCount}
-            onUpdateEntityDetails={fetchDashboard}
-            onUpdateFeedCount={handleFeedCount}
-          />
-        ),
-      },
-      {
-        label: <TabsLabel id={EntityTabs.LINEAGE} name={t('label.lineage')} />,
-        key: EntityTabs.LINEAGE,
-        children: (
-          <LineageProvider>
-            <Lineage
-              deleted={deleted}
-              entity={dashboardDetails as SourceType}
-              entityType={EntityType.DASHBOARD}
-              hasEditAccess={editLineagePermission}
-            />
-          </LineageProvider>
-        ),
-      },
-      {
-        label: (
-          <TabsLabel
-            id={EntityTabs.CUSTOM_PROPERTIES}
-            name={t('label.custom-property-plural')}
-          />
-        ),
-        key: EntityTabs.CUSTOM_PROPERTIES,
-        children: dashboardDetails && (
-          <div className="m-sm">
-            <CustomPropertyTable<EntityType.DASHBOARD>
-              entityDetails={dashboardDetails}
-              entityType={EntityType.DASHBOARD}
-              handleExtensionUpdate={onExtensionUpdate}
-              hasEditAccess={editCustomAttributePermission}
-              hasPermission={viewAllPermission}
-            />
-          </div>
-        ),
-      },
-    ],
-    [
-      feedCount.totalCount,
-      activeTab,
-      isEdit,
-      tableColumn,
-      dashboardDetails,
-      charts,
-      deleted,
-      entityName,
-      dashboardTags,
-      onCancel,
-      handleFeedCount,
-      onDescriptionEdit,
-      onDescriptionUpdate,
-      onThreadLinkSelect,
-      handleTagSelection,
-      editTagsPermission,
-      editGlossaryTermsPermission,
+    const tabs = dashboardDetailsClassBase.getDashboardDetailPageTabs({
       editLineagePermission,
-      editDescriptionPermission,
       editCustomAttributePermission,
-      editAllPermission,
       viewAllPermission,
-    ]
-  );
+      dashboardDetails,
+      deleted: deleted ?? false,
+      handleFeedCount,
+      feedCount,
+      activeTab,
+      getEntityFeedCount,
+      fetchDashboard,
+      labelMap: tabLabelMap,
+    });
+
+    return getDetailsTabWithNewLabel(
+      tabs,
+      customizedPage?.tabs,
+      EntityTabs.DETAILS
+    );
+  }, [
+    feedCount.totalCount,
+    activeTab,
+    dashboardDetails,
+    deleted,
+    handleFeedCount,
+    editLineagePermission,
+    editCustomAttributePermission,
+    editAllPermission,
+    viewAllPermission,
+    onExtensionUpdate,
+  ]);
+
+  const fetchDocument = useCallback(async () => {
+    const pageFQN = `${EntityType.PERSONA}${FQN_SEPARATOR_CHAR}${selectedPersona.fullyQualifiedName}`;
+    try {
+      const doc = await getDocumentByFQN(pageFQN);
+      setCustomizedPage(
+        doc.data?.pages?.find((p: Page) => p.pageType === PageType.Dashboard)
+      );
+    } catch (error) {
+      // fail silent
+    }
+  }, [selectedPersona.fullyQualifiedName]);
+
```

Comment:
In case of failure, we are not setting the customizedPage and by default it's null. should there be a fallback layout in such cases?

Reply:
Yes fallback layout is present for all the base classes. i.e. here: https://github.com/open-metadata/OpenMetadata/blob/ee5f4f5b17bf1c3345ef28be79eae0f2b923f092/openmetadata-ui/src/main/resources/ui/src/components/Customization/GenericTab/GenericTab.tsx#L36